src/test/ui/issues/issue-76042.rs

```diff
@@ -0,0 +1,15 @@
+// run-pass
```

Comment:
I would expect that this that would have a few extra compile flags. `compile-flags: -Coverflow-checks=off -Ccodegen-units=1 -Copt-level=0` with `--target i686-unknown-linux-gnu` also being relevant but omitted.

Reply:
I don't think `--target i686-unknown-linux-gnu` (or rather `only-i686-unknown-linux-gnu`) isn't useful here, this test should pass for every target.
I'll add `compile-flags: -Coverflow-checks=off -Ccodegen-units=1 -Copt-level=0` which I didn't think were needed for the repro. My bad